### PR TITLE
Feature/9 configurable amounts

### PR DESCRIPTION
**Pull request by robbiepaul**
# User need

> As a local authority/housing association
> I need to be able to change the amount we charge for services
> So that we receive the right amount of money from users

# Screenshot 
screen shot 2018-07-20 at 16 12 40

### CODE REVIEW

**Comment by robbiepaul** [comment]
[Configurable £ amounts](https://trello.com/c/Spe7G6zu/9-configurable-amounts)

File: models/penalty_charge_notice.rb [hunk 1]
```diff
@@ -13,9 +13,14 @@ class PenaltyChargeNotice < ActiveRecord::Base
   }.freeze
 
   def charge_amount
+    return CHARGES[charge_type.to_sym] / 2 if discounted?
     CHARGES[charge_type.to_sym]
   end
 
+  def discounted?
+    ((14.days.ago)..(Time.now)).include?(issued_at)
```
Comment (evilstreak) [inline models/penalty_charge_notice.rb]: Would this be clearer as `issued_at < 14.days.ago?`

File: models/penalty_charge_notice.rb [hunk 2]
```diff
@@ -21,6 +21,10 @@ def discounted?
     ((14.days.ago)..(Time.now)).include?(issued_at)
   end
 
+  def description
+    charge_type.humanize + (discounted? ? ' (early payment discount)' : '')
```
Comment (evilstreak) [inline models/penalty_charge_notice.rb]: I'm not sure we need to include the discount on the GOV.UK payment page – showing it on our own page would probably be enough?

**Review by evilstreak — APPROVED** [review]
A couple of minor comments – otherwise looks good 👍

**Pull request merged**